src/styles.css

```diff
@@ -81,4 +81,14 @@ h3.popover-title {
 
 .btn.navbar-text {
   margin-top: 20px;
-}
\ No newline at end of file
+}
+
+.pagination .page-number a {
+  color: #88939a !important;
```

Comment:
Why the !important? Is this due to the way the external style sheet is loaded? If so these should be loaded after in general I would have thought?

Reply:
I can look into this again, e.g. define this as a CSS module just for this component. Also, a general discussion regarding how we use CSS in react applications would be helpful, as there are a number of options, only one of them being scss.